### DISCUSSION TITLE
Does the combined p-values calculated by FindConservedMarkers need adjustment?

### DISCUSSION THREAD

**Discussion by YangLi-Bio** [satijalab/seurat#5401] [General]
Hello, I am using FindConservedMarkers to identify differentially expressed genes (DEGs) by FindConservedMarkers. The DEGs will be filtered according the combined p-values. 

However, I am wondering whether the combined p-values need adjustment?

If needed, what method could be used for adjustment?

Thank you!

**Comment by torkencz** [comment]
Hi, I converted this because I think this is an interesting discussion to have. From what I can see when we are using a method of combining p values from multiple analyses (in this case lets say control vs treatment) we should not do multiple comparison correction afterwards. I based my answer on this: https://stats.stackexchange.com/questions/139311/using-multiple-comparisons-corrections-in-fisher-p-value-framework , but I could be wrong.
